assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -116,7 +117,7 @@ function DisplayElement({ element }) {
           opacity: opacity ? opacity / 100 : null,
         }}
       >
-        <Display element={element} box={box} />
+        <Display element={element} previewMode={previewMode} box={box} />
```

Comment:
Quick favor: the line below should also be `{previewMode && (<ReplacementContainer...>...)}`